kitchen-terraform.gemspec

```diff
@@ -30,22 +41,25 @@ require "rubygems"
   specification.add_development_dependency "middleman-livereload", "~> 3.4"
   specification.add_development_dependency "middleman-syntax", "~> 3.0"
   specification.add_development_dependency "middleman", "~> 4.2"
-  specification.add_development_dependency "mini_racer", "~> 0.3.0"
-  specification.add_development_dependency "pry-coolline", "~> 0.2"
-  specification.add_development_dependency "pry", "~> 0.10"
-  specification.add_development_dependency "reek", "~> 5.5"
   specification.add_development_dependency "rspec", "~> 3.4"
   specification.add_development_dependency "rufo", "~> 0.7"
   specification.add_development_dependency "simplecov", "~> 0.16.1"
   specification.add_development_dependency "travis", "~> 1.8"
+  specification.add_development_dependency "tty-which", "~> 0.4.0"
   specification.add_development_dependency "yard", "~> 0.9"
-  specification.add_runtime_dependency "dry-validation", "~> 0.13"
+  specification.add_development_dependency "dry-validation", "~> 0.13"
+
+  if version_gt_25
+    specification.add_development_dependency "reek", "~> 6.0.2"
+  else
+    puts 'version <= 2.5'
+    specification.add_development_dependency "reek", "~> 5.5"
+  end
+
+  # specification.add_runtime_dependency "mixlib-shellout", "~> 3.0"
```

Comment:
The mixlib-shellout gem is an explicit dependency; it should not be removed.

Reply:
As I recall, it is not a dependency at the top level. With the top level dependencies in the Gemspec, any further child dependencies get auto-installed. All the tests pass, so I think there is nothing lacking in dependencies. I tried to trim the Gemspec dependencies down as much as possible. Windows caused lots of version conflicts.

Reply:
It is in fact a direct dependency for our domain `ShellOut` class: https://github.com/newcontext-oss/kitchen-terraform/blob/0b1dc3ff61397f49767c6131710591c7b63e2176/lib/kitchen/terraform/shell_out.rb#L19

I've had a couple of false starts removing all of that stuff in favour of Kitchen's own shell out logic, but for now it stands. Kitchen also uses mixlib-shellout to implement its shell out logic which is why this removal doesn't break the gem in obvious ways, but without this explicit dependency, things could easily break with unexpected changes to Kitchen.

Reply:
Ok, I uncommented it in the latest push. I hope the Windows testing still passes. If not, I may need to put a conditional in the Gemspec.